src/VoteView.js

```diff
@@ -29,18 +55,30 @@ function VoteView() {
   const sortedCandidates = candidates.sort((a, b) => {
     return b.voteCount - a.voteCount;
   });
+
   return (
     <Container>
       <Title>CEOS 13기 FRONT 운영진 투표 &gt;.0</Title>
-      <Container>순위 이름 득표수</Container>
-      {sortedCandidates.map((candidate) => (
-        <CandidateVotes
-          candidate={candidate}
-          key={candidate.id}
-          flipVoteFlag={flipVoteFlag}
-          rank={sortedCandidates.indexOf(candidate) + 1}
-        />
-      ))}
+      <Link to="/signup">
+        <UserButton r={207} g={216} b={220}>
+          회원가입
+        </UserButton>
+      </Link>
+      <LoginoutButton
+        isLogined={isLogined}
+        handleLogoutButton={handleLogoutButton}
+      ></LoginoutButton>
+      <CandidateContainer>
+        {sortedCandidates.map((candidate) => (
+          <CandidateVotes
+            candidate={candidate}
+            key={candidate.id}
+            flipVoteFlag={flipVoteFlag}
+            rank={sortedCandidates.indexOf(candidate) + 1}
```

Comment:
```suggestion
        {sortedCandidates.map((candidate, index) => (
          <CandidateVotes
            candidate={candidate}
            key={candidate.id}
            flipVoteFlag={flipVoteFlag}
            rank={index + 1}
```

굳이 indexOf를 다시 사용하면 `O(n)`에서 `O(n^2)`이 되니깐 비효율적이겠지?
저런 경우에는 그냥 map 인자로 들어가는 함수 두번째 인자가 index니까 그걸 사용하면 돼.